public/docs/ts/latest/guide/animations.jade

```diff
@@ -208,72 +203,71 @@ figure.image-display
   that the browser considers *animatable*. This includes positions, sizes, transforms, colors,
-  borders and many others. The W3C maintains
-  [a list of animatable properties](https://www.w3.org/TR/css3-transitions/#animatable-properties).
+  borders, and many others. The W3C maintains
+  [a list of animatable properties](https://www.w3.org/TR/css3-transitions/#animatable-properties)
+  on its [CSS Transitions page](https://www.w3.org/TR/css3-transitions).
 
-  For positional properties that have a numeric value, we can define a unit by providing
+  For positional properties that have a numeric value, define a unit by providing
   the value as a string with the appropriate suffix:
 
   * `'50px'`
   * `'3em'`
   * `'100%'`
 
-  For most dimensional properties we can also just define a number which is then assumed to be
-  in pixels:
+  Define most dimensional properties as a number in pixels:
 
   * `50` is the same as saying `'50px'`
 
-  ## Automatic Property Calculation
+  ## Automatic property calculation
 figure
   img(src="/resources/images/devguide/animations/animation_auto.gif" alt="Animation with automated height calculation" align="right" style="width:220px;margin-left:20px" )
 :marked
-  Sometimes the value of a dimensional style property that we want to
-  animate is not known until at runtime. For example, it is quite common for elements
-  to have widths and heights that depend on their content and the screen size. These
-  properties are often tricky to animate with CSS.
+  Sometimes your app doesn't know the value of a dimensional style property until runtime.
+  For example, it is quite common for elements to have widths and heights that
+  depend on their content and the screen size. These properties are often tricky
+  to animate with CSS.
 
-  With Angular we can use a special `*` property value in these cases. What it means
-  is that the value of this property will be computed at runtime and then plugged into
-  the animation.
+  In these cases, you can use a special `*` property value so that the value of the
+  property is computed at runtime and then plugged into the animation.
 
-  The "leave" animation in this example takes whatever height the element has before it
+  In this example, the "leave" animation takes whatever height the element has before it
   leaves and animates from that height to zero:
 
 +makeExample('animations/ts/app/hero-list-auto.component.ts', 'animationdef')(format=".")
 
 :marked
-  ## Animation Timing
+  ## Animation timing
 
-  There are three timing properties we can tune for every animated transition:
-  The duration, the delay, and the easing function. They are all combined into
-  a single transition *timing string*.
+  There are three timing properties you can tune for every animated transition:
+  the duration, the delay, and the easing function. They are all combined into
+  a single transition, *timing string*.
```

Comment:
delete the comma?
(It would seem OK without the italics, but with them it reads oddly to me. Like "transition" and "timing string" are the same thing.)